app/views/admin/deploy_groups/show.html.erb

```diff
@@ -40,7 +40,16 @@
                     class: "btn btn-default"
         %>
         <%= link_to "Deploy All Projects", deploy_all_admin_deploy_group_path(@deploy_group), class: "btn btn-default", data: { method: "post" } %>
-      <% end %>
+        <%= link_to "Merge These Stages",
+                    merge_all_stages_admin_deploy_group_path(@deploy_group),
+                    class: "btn btn-default",
+                    data: {
+                      method: "post",
+                      confirm: "All stages listed will be deleted, and this Deploy Group will be added to the template stage for each project.\n\n" +
```

Comment:
The UI only shows "all stages that use this deploy group" ... and this button only affects "stages that have exactly this deploy group and are on a project that uses include_new_deploy_groups"